media/js/base/stub-attribution.js

```diff
@@ -395,7 +414,8 @@ if (typeof window.Mozilla === 'undefined') {
             ua: ua,
             experiment: experiment,
             variation: variation,
-            visit_id: visitID
+            visit_id: clientID, // passed through as visit_id as was originally named incorrectly
```

Comment:
The stub service now supports `client_id` in addition to `visit_id`.

Reply:
@willdurand thanks! I've added a commit which updated bedrock to pass `client_id`. I'll push this to demo as well.